client/pages/home.jsx

```diff
@@ -1,8 +1,120 @@
-import React from 'react';
+/* eslint-disable multiline-ternary */
+
+import React, { useEffect, useState } from 'react';
+
+const App = () => {
+  const [venues, setVenues] = useState([]);
+  const [searchText, setSearchText] = useState('');
+  const [searchCity, setSearchCity] = useState('');
+  const [imageLinks, setImageLinks] = useState([]);
+  const baseUrl = 'http://localhost:3001';
+
+  useEffect(() => {
+    // eslint-disable-next-line array-callback-return
+    venues.map((venue, i) => {
+
+      if (i > 1) return null;
+
+      fetch(`${baseUrl}/api/venue/image/${venue.id}`)
+        .then(data => data.json())
+        .then(res => {
+          imageLinks.push({ id: venue.id, url: res.imgUrl });
+
+          const copiedImageLinks = JSON.parse(JSON.stringify(imageLinks));
```

Comment:
Instead of making a clone like this, you can make a new array with `Array.prototype.concat()`.

```js
const newImageLinks = imageLinks.concat({ id: venue.id, url: res.imgUrl });
```